getting_started/workflow/assets/escn_exporter/skeleton.rst

```diff
@@ -0,0 +1,32 @@
+Skeleton
+========
+
+Rest Bone
+---------
+
+The Armature object in Blender is exported as a Skeleton node.
+The rest position and parent relation of bones are exported:
+
+.. image:: img/armature.jpg
+
+.. warning:: 
+    The three check boxes :code:`Inheirt Rotation`, :code:`Inheirt Scale`,
+    :code:`Local Location` (colored in red) must be ticked when building
+    armature in Blender, so that the exported bone transform would be
+    consistant between Blender and Godot
+
+Non-Deform Bone
+---------------
+
+Note that the non-deform bone can be configured as not exported
+by enable the :code:`Exclude Control Bones`, the deform bone
+checkbox in Blender is there:
+
+.. image:: img/armature_deform_bone.jpg
+
+Bone Attachment
+---------------
+Export the parenting relation between a bone and an object
+is supported:
+
+.. image:: img/armature_bone_attachment.jpg
```

Comment:
I'm not sure if this needs an image, because parenting relationships are set up with the keyboard shortcut. I would be tempted to have only a single image for all of these three headings (the first two images are quite similar)